AzureRmStorageTableCoreHelper.psm1

```diff
@@ -520,6 +526,14 @@ function Get-AzTableRow
 		[Parameter(ParameterSetName="byCustomFilter")]
 		$Table,
 
+		[Parameter(ParameterSetName="GetAll")]
+		[Parameter(ParameterSetName="byPartitionKey")]
+		[Parameter(ParameterSetName="byPartRowKeys")]
+		[Parameter(ParameterSetName="byColummnString")]
+		[Parameter(ParameterSetName="byColummnGuid")]
+		[Parameter(ParameterSetName="byCustomFilter")]
+		[array]$SelectColumns,

```

Comment:
We can use `[System.Collections.Generic.List[string]]$SelectColumn` and shorten the changes a little bit, this will accept being called the same way as if array was defined.